### PR TITLE
Full dashboard

### PR DESCRIPTION
**Pull request by oninach**
- The dashboard can now display all visualizations on a single page.
- The button on the control panel allows you to select the one you want.
- The visualization is resizable

### CODE REVIEW

**Review by motrell — APPROVED** [review]
In general, I'm fine with this. There are some issues with it (see below), but it seems to be not critical at all.

We did not do, what the ticket expected, because we just changed the visualization of the visualizations without using OS-Dashboard-Dashboards. But I really like the new design. We should talk about in the team meeting, whether we want explicitly the OS-Dashboard-Dashboards.

File: src/application/res/templates/visualizations.html
[the remark this answers is not in the document]
Comment (oninach) [reply src/application/res/templates/visualizations.html:1]: Yes I know where the problem is coming from ))
We will see first if this work make sense for our expectation, if yes we can fix that ))

File: src/application/res/backend/get_all_iframes.py
[the remark this answers is not in the document]
Comment (oninach) [reply src/application/res/backend/get_all_iframes.py:1]: No, I just forgot to delete that...

File: src/application/res/templates/visualizations.html
[the remark this answers is not in the document]
Comment (motrell) [reply src/application/res/templates/visualizations.html:1]: Yes, of course. Maybe we show this to @tawfeeka and @lukas-rotsching , so they can think about create a bug report.

File: src/application/res/backend/get_all_iframes.py
[the remark this answers is not in the document]
Comment (motrell) [reply src/application/res/backend/get_all_iframes.py:1]: Alright

**Review by leondaniel22 — APPROVED** [review]

**Review by Amir-Hussein-OTH — APPROVED** [review]

**Pull request merged**